### PR TITLE
Close #227: preparing sample use cases for a demo and to show how to use the clustered management layer

### LINKED ISSUE #227: Add a little M&M sample
that could be used as a startup doc

### PR DESCRIPTION
**Pull request by mathieucarbou**
FYI @henri-tremblay : you were asking how to use the clustered management layer... Here are some samples.

### CODE REVIEW

**Comment by mathieucarbou** [comment]
UPDATE: @anthonydahanne @GaryWKeim : this PR is done, complete and tested with a real server. So you can merge.

Also, it depends on PR #235 : so either you directly merge this (it will close #235 also), or you do it separately. It's the same.

File: management/testing/doc/src/main/java/org/terracotta/management/doc/ReadStatistics.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        .filter(c -> c.getName().startsWith("Ehcache:"))
+        .forEach(ehcache -> {
+
+          // 2. create a routing context
+          Context ctx = ehcache.getContext()
+              .with("cacheManagerName", "my-super-cache-manager");
+
+          try {
+            // 3. collect stats on client-side
+            tmsAgentService.updateCollectedStatistics(ctx, "StatisticsCapability", asList("Cache:HitCount", "Clustered:HitCount", "Cache:MissCount", "Clustered:MissCount")).waitForReturn();
```
Comment (anthonydahanne) [inline management/testing/doc/src/main/java/org/terracotta/management/doc/ReadStatistics.java]: hum, you assume here that an ehcache client is connected and running.
that's fine, but that's not part of the readme though.

File: management/testing/doc/README.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    <plugins>
+      <config>
+        <ohr:offheap-resources>
+          <ohr:resource name="primary" unit="MB">64</ohr:resource>
+        </ohr:offheap-resources>
+      </config>
+    </plugins>
+  </tc-config>
+
+5. Go to `server/bin` and run: `./start-tc-server.sh -f tc-config.xml`
```
Comment (mathieucarbou) [inline management/testing/doc/README.adoc]: I do not want to have server files or ehcache files in the tc-platform project... It is juts for demo so if we start putting some files, I fear we will continue and create a dependency over other projects... :-S

File: management/testing/doc/src/main/java/org/terracotta/management/doc/ReadStatistics.java (hunk 1 above)
Comment (mathieucarbou) [inline management/testing/doc/src/main/java/org/terracotta/management/doc/ReadStatistics.java]: no I do not assume that and I tested them without an ehcache client, they work. BUT if some ehcache clients are there, I will also collect their stats.

README: `For some, you might need to have a cache manager initialized first.`

**Comment by mathieucarbou** [comment]
@anthonydahanne : as I already said (emails and scrums) this is just a standalone example to show to use the management api to query remotely management information of a cluster because:
1. we had some people internally asking for how the m&m stack can be used
2. we had now something to demo to show how it works.
It is not intended for blog post or other external use since M&M is not official.

**Pull request merged**